packages/syntax-core/src/RadioButton/RadioButton.tsx

```diff
@@ -71,7 +71,7 @@ const RadioButton = ({
   /**
    * Value of the selected radio option
    */
-  value: string;
+  value?: string | number;
```

Comment:
moving my comment to here: 

should value ever be optional? could you update this before merging?

Reply:
alright I made it non-optional again